bots/tests-scan

```diff
@@ -69,7 +69,16 @@ REPO_BRANCH_CONTEXT = {
     },
     'weldr/lorax': {
         'master': [
-            'cockpit/fedora-30',
+            'cockpit/fedora-30/TestImages.test_ext4_filesystem',
+            'cockpit/fedora-30/TestImages.test_partitioned_disk',
+            'cockpit/fedora-30/TestImages.test_tar',
+            'cockpit/fedora-30/TestImages.test_qcow2',
+            'cockpit/fedora-30/TestImages.test_live_iso',
+
+            'cockpit/fedora-30/TestCloud.test_aws',
+            'cockpit/fedora-30/TestCloud.test_azure',
+            'cockpit/fedora-30/TestCloud.test_openstack',
+            'cockpit/fedora-30/TestCloud.test_vmware',
```

Comment:
`test/run` in lorax only uses the last part as `$TEST_SCENARIO`. Whatever we do, I wouldn't put python class names in here. They're needlessly long.

Reply:
That will mean all test methods should be in the same class.